src/main/java/htsjdk/samtools/DuplicateScoringStrategy.java

```diff
@@ -64,6 +65,8 @@ public static short computeDuplicateScore(final SAMRecord record, final ScoringS
 
     /**
      * Returns the duplicate score computed from the given fragment.
+     * value should be capped by Short.MAX_VALUE since the score from two reads will be
```

Comment:
and then what? the sorting collection holds a short and I don't want to pay the memory cost of changing that to an int...